### ISSUE TITLE
:farmer: Rolling performance job failing due to change in Fast-CDR version

### ISSUE THREAD

**Issue by Crola1702** [ros2/performance_test#43]
Last week, rolling performance job has been failing, this is due to a change in Fast-CDR version.

Passing builds used to have version 1.1.x, but newer ones use 2.2.x

Reference build: 
* https://build.ros2.org/view/Rci/job/Rci__nightly-performance_ubuntu_jammy_amd64/664/ (failing downstream)
* https://build.ros2.org/job/Rci__nightly-extra-rmw-release_ubuntu_jammy_amd64/782/ (passing upstream)

Checking the upstream builds, the comparebuildrepos diff is:

<details>
<summary>
repos diff
</summary>

```diff
@@ -31,11 +31,11 @@ repositories:
   eProsima/Fast-CDR:
     type: git
     url: https://github.com/eProsima/Fast-CDR.git
-    version: bdceab51521232d66dc11b0e9dbc79cb2762fd4e
+    version: f44cbf67c10dc3438e9dd96ded8d1b28c9abc075
   eProsima/Fast-DDS:
     type: git
     url: https://github.com/eProsima/Fast-DDS.git
-    version: 1d9d559b2293fd3a1e197e0b8ec732833d76a6a6
+    version: 33eb8be4b42d735e84f1754bfecaa6f294e153b1
   eProsima/foonathan_memory_vendor:
     type: git
     url: https://github.com/eProsima/foonathan_memory_vendor.git
@@ -43,7 +43,7 @@ repositories:
   eclipse-cyclonedds/cyclonedds:
     type: git
     url: https://github.com/eclipse-cyclonedds/cyclonedds.git
-    version: 84b8713f57e54780a93883b647074b41fdfe1ced
+    version: b47d42fd976e96cf0833b5cbb2371d82970bebaf
   eclipse-iceoryx/iceoryx:
     type: git
     url: https://github.com/eclipse-iceoryx/iceoryx.git
@@ -67,7 +67,7 @@ repositories:
   ros-perception/image_common:
     type: git
     url: https://github.com/ros-perception/image_common.git
-    version: feab1730aab853dada9811717732f2a44889b9c6
+    version: daded2efc366e805bc0277b841cfadfe6ed8d56f
   ros-perception/laser_geometry:
     type: git
     url: https://github.com/ros-perception/laser_geometry.git
@@ -87,7 +87,7 @@ repositories:
   ros-tooling/libstatistics_collector:
     type: git
     url: https://github.com/ros-tooling/libstatistics_collector.git
-    version: 8d45a4e1c4948c797e5f65d9419787b51b9c0471
+    version: e2d39a62b8e96991b3e7be75a03d6d1d1dcb97bf
   ros-visualization/interactive_markers:
     type: git
     url: https://github.com/ros-visualization/interactive_markers.git
@@ -111,7 +111,7 @@ repositories:
   ros-visualization/rqt_bag:
     type: git
     url: https://github.com/ros-visualization/rqt_bag.git
-    version: 38f1d05c553eaaace0e774c72b832206285faec3
+    version: 8e884e6cc2a20b836a0c7dcccd8bf4d4be5e3491
   ros-visualization/rqt_console:
     type: git
     url: https://github.com/ros-visualization/rqt_console.git
@@ -127,7 +127,7 @@ repositories:
   ros-visualization/rqt_plot:
     type: git
     url: https://github.com/ros-visualization/rqt_plot.git
-    version: 7e46bc5395320b23aa7a3c9b45cfa81237f5b22f
+    version: aa3ec1914d3f510055e6e87968c63fbe85cd064a
   ros-visualization/rqt_publisher:
     type: git
     url: https://github.com/ros-visualization/rqt_publisher.git
@@ -227,7 +227,7 @@ repositories:
   ros2/geometry2:
     type: git
     url: https://github.com/ros2/geometry2.git
-    version: 8752526562dac12e9fc87ba1e7a675ef6ec07c35
+    version: 0e505aaf09bac77242b483fa8a8ede5047a0ddc4
   ros2/launch:
     type: git
     url: https://github.com/ros2/launch.git
@@ -235,7 +235,7 @@ repositories:
   ros2/launch_ros:
     type: git
     url: https://github.com/ros2/launch_ros.git
-    version: 862520df5e39ad0d469a1aabb1e0196cfad70da8
+    version: 5dd06c92e041571df866103c6f664c8dd0d60898
   ros2/libyaml_vendor:
     type: git
     url: https://github.com/ros2/libyaml_vendor.git
@@ -267,7 +267,7 @@ repositories:
   ros2/rcl:
     type: git
     url: https://github.com/ros2/rcl.git
-    version: ae0296f43ae4951f1226818ae61ea955f882530c
+    version: 56957638909e5aeab8ba3ac8797d4c5484efb895
   ros2/rcl_interfaces:
     type: git
     url: https://github.com/ros2/rcl_interfaces.git
@@ -279,11 +279,11 @@ repositories:
   ros2/rclcpp:
     type: git
     url: https://github.com/ros2/rclcpp.git
-    version: b50f9ab41840dfec121b7d12118da6e6593d21ae
+    version: 42810ee3be16621208e2439cd6257a584d827759
   ros2/rclpy:
     type: git
     url: https://github.com/ros2/rclpy.git
-    version: 7d52cfa9801597c62be98f80d1a2c769f9768fff
+    version: a3ecd7e84bc3bcce0e5935d50744d2a161669a40
   ros2/rcpputils:
     type: git
     url: https://github.com/ros2/rcpputils.git
@@ -303,7 +303,7 @@ repositories:
   ros2/rmw_connextdds:
     type: git
     url: https://github.com/ros2/rmw_connextdds.git
-    version: a127514d2e480f59056c9568cf9ae9b94ae2f1cc
+    version: e11d404d860c876e15e1bff63fcd026512732729
   ros2/rmw_cyclonedds:
     type: git
     url: https://github.com/ros2/rmw_cyclonedds.git
@@ -315,7 +315,7 @@ repositories:
   ros2/rmw_fastrtps:
     type: git
     url: https://github.com/ros2/rmw_fastrtps.git
-    version: 829cde3d466d0c00bcc809cd0cbe323918b978f8
+    version: 2b3257bddf47b7ca2fad921ec03cd472a22588ae
   ros2/rmw_implementation:
     type: git
     url: https://github.com/ros2/rmw_implementation.git
@@ -323,7 +323,7 @@ repositories:
   ros2/ros2_tracing:
     type: git
     url: https://github.com/ros2/ros2_tracing.git
-    version: 115034b9ec00b9667bc8d98b48472d73c2b9f8b6
+    version: 0d720e162b8cc85852b5109f551c57c1ade7301b
   ros2/ros2cli:
     type: git
     url: https://github.com/ros2/ros2cli.git
@@ -343,7 +343,7 @@ repositories:
   ros2/rosidl:
     type: git
     url: https://github.com/ros2/rosidl.git
-    version: ad59de05bdeddf7115021445007d76ef6dd531ff
+    version: bbddf4671dc536e5af0cec7fcbc785253f124d54
   ros2/rosidl_core:
     type: git
     url: https://github.com/ros2/rosidl_core.git
@@ -371,7 +371,7 @@ repositories:
   ros2/rosidl_runtime_py:
     type: git
     url: https://github.com/ros2/rosidl_runtime_py.git
-    version: 82af486e09d5463c2c4cc5135d2e122735d4f2be
+    version: 2e831a241c8655cd015301d9e1d3c7f09e84a0bc
   ros2/rosidl_typesupport:
     type: git
     url: https://github.com/ros2/rosidl_typesupport.git
@@ -379,7 +379,7 @@ repositories:
   ros2/rosidl_typesupport_fastrtps:
     type: git
     url: https://github.com/ros2/rosidl_typesupport_fastrtps.git
-    version: b4f63c02453b8a8afb5e996229d309cb6d1c41f5
+    version: edf7835187403fdb40f9d97c6bb904499157cf83
   ros2/rpyutils:
     type: git
     url: https://github.com/ros2/rpyutils.git
@@ -387,7 +387,7 @@ repositories:
   ros2/rviz:
     type: git
     url: https://github.com/ros2/rviz.git
-    version: 4ad1eba618fc1d5e11d0319f6648d484daeb3008
+    version: f3f8c1f8e3c801c5c2eb4176341fde299b50bba1
   ros2/spdlog_vendor:
     type: git
     url: https://github.com/ros2/spdlog_vendor.git
@@ -399,7 +399,7 @@ repositories:
   ros2/system_tests:
     type: git
     url: https://github.com/ros2/system_tests.git
-    version: 243e4913dc552eb8c6a4e67dd3648ea4ddecee2e
+    version: 76057be33c56baa0a401582e4e3fddfd106db61e
   ros2/test_interface_files:
     type: git
     url: https://github.com/ros2/test_interface_files.git
```

</details>

In Fast-CDR changes we see a change of branch! from `1.1.x` -> `2.2.x`

[1.1.x commit](https://github.com/eProsima/Fast-CDR/commit/bdceab51521232d66dc11b0e9dbc79cb2762fd4e) to [2.2.x commit](https://github.com/eProsima/Fast-CDR/commit/f44cbf67c10dc3438e9dd96ded8d1b28c9abc075)

**Comment by MiguelCompany** [comment]
@Crola1702 @clalancette See #45 which should fix this

**Comment by clalancette** [comment]
This was fixed by #45, so closing.
